### PR TITLE
Update passwordRegex

### PR DESCRIPTION
**Pull request by HuckMatt**
Zoom invite uses 'Passcode' to describe access code.

### CODE REVIEW

**Comment by HuckMatt** [comment]
@eric-giannini please review

File: README.md [hunk 1]
```diff
@@ -1,52 +1,109 @@
...
 
-## References
-
-Feel free to fork any of the projects here to use on your own and if you have any code improvements, make a pull request and the request will be reviewed by one of our admins.
-
-For a sample reference implentation using Symbl, take a look at our [Platform](https://platform.symbl.ai).
+## Community
 
-If you have questions, bugs to report or feature suggestions, join our [Dev Community](https://community.symbl.ai/).
+If you liked our integration guide, please star our repo! If you have any questions, feel free to reach out to us at devrelations@symbl.ai or through our Community Slack at https://developer.symbl.ai/community/slack or our [developer community][developer_community].
```
Comment (eric-giannini) [inline README.md:104]: @HuckMatt Please reformat. See below. 

Screen Shot 2021-01-22 at 3 58 34 PM

File: README.md [hunk 2]
```diff
@@ -1,52 +1,109 @@
 # Symbl Personal Assistant Meeting App
```
Comment (eric-giannini) [inline README.md:1]: Please add respective covenants.

File: README.md (hunk 1 above)
Comment (HuckMatt) [reply README.md]: @eric-giannini reformatted in latest commit

File: README.md (hunk 2 above)
Comment (HuckMatt) [reply README.md:1]: @eric-giannini covenant added in latest commit

**Pull request merged**